docusaurus/docs/guides/Public_Cloud_Deployment/Controller.md

```diff
@@ -229,6 +395,48 @@ export EXTERNAL_DNS=$(dig +short -x $(curl -s icanhazip.com) | sed "s/.$//")
 ```
 Then follow the [Host OpenZiti Anywhere](/docs/learn/quickstarts/network/hosted/#express-install) guide to setup the controller.
 
+</TabItem>
+<TabItem value="DigitalOcean">
+
+- Once the VM is created, we can get the IP address of the droplet from the Resources screen.
+- Login to the VM by using user "root" and IP address:
+```bash
+ssh root@<ip>
+```
+Then follow the [Host OpenZiti Anywhere](/docs/learn/quickstarts/network/hosted/) to setup the controller. You must replace the EXTERNAL_DNS with the following command before running the quickstart.
+
+- **export EXTERNAL_DNS="$(curl -s eth0.me)"**
```

Comment:
@gberl002 - will things work properly with EXTERNAL_DNS set to an IP address??? That surprises me if that works out still?

Reply:
Yes, it works. Obviously it _should_ be the DNS but if it's set to IP then the IP is used for advertising instead of the DNS, as far as the PKI, the IP will only be added as an IP in the SANs.

Reply:
Does that mean the OVERRIDE_IP's all need to be set too ? I'm not entirely sure what you mean "if it's set to IP"?

Reply:
I was assuming someone was following the host it anywhere quickstart. The install would not work unless all the environment variables were set as expected in the "Host it Anywhere" quickstart.

So, yes, the ..._IP_OVERRIDE values would all need to be set as well. Setting just EXTERNAL_DNS would not work with an IP address.

Reply:
Right, I don't see that set in here. @jlin-nf - you verified traffic passes through this setup, on all the platforms, right?  It seems like the PKI will maybe be wrong... It's just a bit worrisome to me is all. I just want to make sure someone tested traffic passing

Reply:
@dovholuknf Yes I did.  I still have a system that is on the "shutdown" state, you are welcome to inspect.  
The overwite value geoff mentioned above:
export EXTERNAL_IP="$(curl -s eth0.me)"       
export ZITI_CTRL_EDGE_IP_OVERRIDE="${EXTERNAL_IP}"
export ZITI_ROUTER_IP_OVERRIDE="${EXTERNAL_IP}"
export ZITI_CTRL_EDGE_ADVERTISED_ADDRESS="${EXTERNAL_DNS:-${EXTERNAL_IP}}"
export ZITI_ROUTER_ADVERTISED_HOST="${EXTERNAL_DNS:-${EXTERNAL_IP}}"
export ZITI_CTRL_ADVERTISED_PORT=8440
export ZITI_CTRL_EDGE_ADVERTISED_PORT=8441
export ZITI_ROUTER_PORT=8442

are still needed. Since that is mentioned in the quick start guide already, i didn't re-mention them.

Reply:
I'm specifically asking about setting EXTERNAL_DNS though. Your example block you posted does not set EXTERNAL_DNS as is done in the PR, it is setting the EXTERNAL_IP. That makes me feel like you need to retest/reverify that you actually set EXTERNAL_DNS too. See what I mean?

Reply:
The EXTERNAL_DNS has to be set to the external IP as I mentioned in the document.  Then the rest of setting are documented in the quickstart guide. (which I skip over for not duplicate the document).  They all needed for the external IP to be advertised correctly.

Reply:
The test I performed is with the following environment set:

export EXTERNAL_DNS="$(curl -s eth0.me)"
export EXTERNAL_IP="$(curl -s eth0.me)"
export ZITI_CTRL_EDGE_IP_OVERRIDE="${EXTERNAL_IP}"
export ZITI_ROUTER_IP_OVERRIDE="${EXTERNAL_IP}"
export ZITI_CTRL_EDGE_ADVERTISED_ADDRESS="${EXTERNAL_DNS:-${EXTERNAL_IP}}"
export ZITI_ROUTER_ADVERTISED_HOST="${EXTERNAL_DNS:-${EXTERNAL_IP}}"
export ZITI_CTRL_ADVERTISED_PORT=8440
export ZITI_CTRL_EDGE_ADVERTISED_PORT=8441
export ZITI_ROUTER_PORT=8442

Regards

Reply:
@dovholuknf James has tested the above and it is working.  Are you ok with publishing?  If not please provide alternative config that you would like to see.

Reply:
yep. i just happened to notice it and was wondering about the dns/ip apparent conflict.